extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/drive/DriveImporter.java

```diff
@@ -153,4 +156,10 @@ private synchronized Drive getDriveInterface(TokensAndUrlAuthData authData) {
 
     return driveInterface;
   }
+
+  private String getRootFolderName() {
+    return String.format(
+            ROOT_FOLDER_FORMAT_STRING,
+            new SimpleDateFormat("yyyy-MM-dd-HH-mm-ss").format(new Date()));
```

Comment:
In theory we could use the created timestamp from the job to give us a more stable date. Is there a reason we didn't go down that route?

Reply:
oh, good point!